pages/configuration/resin-base-images.md

```diff
@@ -1,8 +1,65 @@
 # Resin Images Wiki
 
-This wiki page contains all information about images on Resin registry.
+This page contains all the information about the image maintained on the Resin.io docker hub registry. 
 
-## Base Images:
+## <a name="image-tree"></a>Resin Image Trees
+
+This section describes the Resin image trees (hierarchy of images). These image trees provide an overview of how all the resin base images fit together for each device type supported by Resin.
+
+__Note:__ In the tree diagram, from the bottom to the top, the lower level image is used as the base docker image to build the upper level one.
+
+### ARMv6: Raspberry Pi 1 (version B, A+ and B+)
+
+![ARMv6 Tree Diagram](/img/armv6-diagram.jpg)
+
+| Image | Description | Installed Packages | Repository | Available Tag|
+|:-----------|:------------|:------------|:------------|:------------|
+| resin/rpi-raspbian | The base OS image for ARMv6 devices (Raspberry Pi 1 - RPI1) | minbase, sudo | [dockerhub][rpi-dockerhub-link] | latest, jessie, wheezy |
+| resin/raspberrypi-systemd | The image with full systemd init system installed for ARMv6 | systemd | [dockerhub][rpi-systemd-dockerhub-link] | latest, jessie, wheezy | 
```

Comment:
@nghiant2710 it may be cool to link from systemd image description down to the "tips" section on how to enable systemd. Something like:
"See our [tips][link-to-tips] section on how to enable systemd in your image.